test/personalization.js

```diff
@@ -63,3 +67,63 @@ describe(`personalization cases`, () => {
 
     });
 });
+
+describe('method to return the correct tagline translation according to the mapping id', () => {
```

Comment:
@lucasfrotam Let's keep the personalization test just about the two public interfaces and move all other tests related to experiments in their own test folder with experiment name.